x-pack/plugin/fleet/build.gradle

```diff
@@ -29,4 +29,5 @@ testClusters.configureEach {
   setting 'xpack.security.enabled', 'true'
   setting 'xpack.security.autoconfiguration.enabled', 'false'
   user username: 'x_pack_rest_user', password: 'x-pack-test-password'
+  systemProperty 'es.queryable_built_in_roles_enabled', 'false'
```

Comment:
Disabling the feature to avoid having to adjust the `FleetDataStreamIT` and handle warnings caused by accessing and deleting `.security` index. I will adjust the `FleetDataStreamIT` in a followup PR.

Reply:
This is not expected to impact production clusters, right? Since they don't interact with the `.security` index directly?